nectar/src/swap/bitcoin.rs

```diff
@@ -88,15 +88,15 @@ impl hbit::ExecuteRefund for Wallet {
 
         let refund_address = self.inner.new_address().await?;
 
-        let fee_rate_per_byte = self.fee.byte_rate().await?;
+        let fee_rate_per_vbyte = self.fee.vbyte_rate().await?;
```

Comment:
Similar to the patch on the HTTP API module, I think we can shorten this variable name without loosing any information to something like `btc_per_vbyte` or `sats_per_vbyte`.

Alternatively, if the method is named `vbyte_rate`, why can't the variable be called the same? :)